src/app.ts

```diff
@@ -1,4 +1,13 @@
-import express from 'express'
+const express = require('express')
+const fs = require('fs')
+const bodyParser = require('body-parser')
```

Comment:
express comes with its own bodyparser, so this can be removed:
`app.use(express.json())` or `app.use(express.urlencoded())`